utils/perl5lib/compilers_translation_tool.pl

```diff
@@ -0,0 +1,575 @@
+#!/usr/bin/env perl
+
```

Comment:
It would be nice to have a description here of what this tool does and the arguments it accepts.